src/test/java/models/CustomerModel.java

```diff
@@ -0,0 +1,43 @@
+package models;
+
+import com.github.javafaker.Faker;
+import lombok.*;
+
+import java.util.Locale;
+
+@Getter
+@Setter
+
+public class CustomerModel {
+
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String address;
+    private String city;
+    private String state;
+    private String zipCode;
+    private final String countryDefault = "United Kingdom"; // You can set the default country here
+    private String login;
+    private String password;
+
+    Faker fakerEN = Faker.instance(new Locale("en-gb"));
+
+    public CustomerModel() {
+        //Generate data pack
+        this.generateData();
+    }
+
+    public void generateData() {
```

Comment:
Use Lombok builder feature here.  Maybe we can rename the method to better describe what it does, something like:
generateCustomerWithRandomData()

Reply:
Ok, I'll change it. 
You are right, that is better name fot the method.